pkg/watches/watches.go

```diff
@@ -55,42 +74,63 @@ func Load(path string) ([]Watch, error) {
 		return nil, err
 	}
 
-	watchesMap := make(map[schema.GroupVersionKind]Watch)
+	watchesMap := make(map[schema.GroupVersionKind]struct{})
 	for i, w := range watches {
-		if err := verifyGVK(w.GroupVersionKind); err != nil {
-			return nil, fmt.Errorf("invalid GVK: %s: %w", w.GroupVersionKind, err)
+		gvk := w.GroupVersionKind
+
+		if err := verifyGVK(gvk); err != nil {
+			return nil, fmt.Errorf("invalid GVK: %s: %w", gvk, err)
+		}
+
+		if _, err := chartutil.IsChartDir(w.ChartPath); err != nil {
+			return nil, fmt.Errorf("invalid chart directory %s: %w", w.ChartPath, err)
 		}
```

Comment:
I don't think we need this check unless we want to prohibit loading from a chart.tar.gz file.

The following `loader.Load()` function will handle figuring out what the path is an return an appropriate error if the passed path is unable to be loaded as a chart.